dgraph/cmd/bulk/run.go

```diff
@@ -115,21 +115,22 @@ func init() {
 		"Ignore UIDs in load files and assign new ones.")
 
 	// Options around how to set up Badger.
-	flag.String("badger.compression", "snappy",
-		"[none, zstd:level, snappy] Specifies the compression algorithm and the compression"+
-			"level (if applicable) for the postings directory. none would disable compression,"+
-			" while zstd:1 would set zstd compression at level 1.")
-	flag.Int64("badger.cache_mb", 64, "Total size of cache (in MB) per shard in reducer.")
-	flag.String("badger.cache_percentage", "70,30",
-		"Cache percentages summing up to 100 for various caches"+
-			" (FORMAT: BlockCacheSize, IndexCacheSize).")
+	flag.String("badger", worker.BadgerDefaults,
+		`Various badger options.
+	compression=[none, zstd:level, snappy] specifies the compression algorithm and the compression
+		level (if applicable) for the postings directory. "none" would disable compression, while
+		"zstd:1" would set zstd compression at level 1.
+	cache_mb=N total size of cache (in MB) per shard in the reducer.
```

Comment:
We need this `cache_mb` flag in Bulk for [setting the Badger cache size](https://github.com/dgraph-io/dgraph/pull/7387/files#diff-76237e26c3354ceb21e16c8c34877ddb485e468d4eb711ba35477f35f9018e96R170). The default value is set in [`worker/server_state.go`](https://github.com/dgraph-io/dgraph/pull/7387/files#diff-02ceb0db5e4dd0958bdba03c2fdee17011b75a2fc34cdaec7d232a828d7bfda8R34).